pmd-java/src/main/java/net/sourceforge/pmd/lang/java/typeresolution/ClassTypeResolver.java

```diff
@@ -1246,6 +1251,13 @@ private void populateType(TypeNode node, String className, int arrayDimens) {
                 myType = pmdClassLoader.loadClass(qualifiedNameInner);
             } catch (Exception ignored) {
```

Comment:
We should probable catch here just "ClassNotFoundException", too - at least, to make it consistent...

Reply:
possibly, yeah!